### PR TITLE
run bash in interactive mode

### LINKED ISSUE #16: autostart shell script does not source bashrc
also it cannot source bashrc, rather just reuse the environment variables from the bashrc and add it to the autostart shell script

**CC:** @neopar is there a work around? 

https://askubuntu.com/questions/64387/cannot-successfully-source-bashrc-from-a-shell-script

### PR DESCRIPTION
**Pull request by neopar**
This is one way to fix #16 as we actually want an interactive shell running.

### CODE REVIEW

**Comment by neopar** [comment]
The other solution would be adding `source ~/.bashrc` to the autostart script.

**Comment by padhupradheep** [comment]
For some reason, this solution didn't work on the mp-500, and I had to use the back up.. 
I can reproduce the issue in MP-400 as well.. if you tried to restart the robot pc, you can see that..

**Comment by padhupradheep** [comment]
Ah.. got it! You need to use `xterm` and not `xfce`..

File: generic/startROS.desktop [hunk 1]
```diff
@@ -3,7 +3,7 @@ Version=1.0
 Type=Application
 Name=startROS
 Comment=
-Exec=xfce4-terminal -H -e '/bin/bash -c "~/ROS_AUTOSTART.sh"'
+Exec=xfce4-terminal -H -e '/bin/bash -i -c "~/ROS_AUTOSTART.sh"'
```
Comment (padhupradheep) [inline generic/startROS.desktop:6]: we should switch to xterm

**Review by padhupradheep — CHANGES_REQUESTED** [review]

**Comment by padhupradheep** [comment]
okay, found it.. you have to add it to this file as well 

https://github.com/neobotix/robot-setup-tool/blob/ros2/generic/ROS-Neobotix-Autostart.desktop 

autostart at desktop works fine..

**Review by padhupradheep — APPROVED** [review]

**Pull request merged**